bftengine/tests/simpleKVBCTests/TesterReplica/main.cpp

```diff
@@ -48,14 +48,6 @@ ReplicaParams rp;
 concordlogger::Logger replicaLogger =
 		concordlogger::Logger::getLogger("skvbctest.replica");
 
-void signalHandler( int signum ) {
-	if(r)
-		r->stop();
-
-	LOG_INFO(replicaLogger, "replica " << rp.replicaId << " stopped");
-	exit(0);
-}
-

```

Comment:
I think we do support a graceful replica shutdown, so I think it's important to allow it shutting down nicely. 
I did see errors during replica stop, but not recursive ones. In my code, I just removed signal(SIGKILL, signalHandler), as a non-catchable.